pandas/tests/io/test_excel.py

```diff
@@ -169,54 +169,155 @@ def test_parse_cols_int(self):
 
         dfref = self.get_csv_refdf('test1')
         dfref = dfref.reindex(columns=['A', 'B', 'C'])
-        df1 = self.get_exceldf('test1', 'Sheet1', index_col=0, parse_cols=3)
-        df2 = self.get_exceldf('test1', 'Sheet2', skiprows=[1], index_col=0,
-                               parse_cols=3)
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
+            df1 = self.get_exceldf('test1', 'Sheet1', index_col=0,
+                                   parse_cols=3)
+            df2 = self.get_exceldf('test1', 'Sheet2', skiprows=[1],
+                                   index_col=0, parse_cols=3)
         # TODO add index to xls file)
         tm.assert_frame_equal(df1, dfref, check_names=False)
         tm.assert_frame_equal(df2, dfref, check_names=False)
 
+    def test_usecols_int(self):
+        # GH4988: inconsistent naming convention for read_excel column select
+        dfref = self.get_csv_refdf('test1')
+        dfref = dfref.reindex(columns=['A', 'B', 'C'])
+        df1 = self.get_exceldf('test1', 'Sheet1', index_col=0, usecols=3)
+        df2 = self.get_exceldf('test1', 'Sheet2', skiprows=[1], index_col=0,
```

Comment:
so you need to change ALL tests to use the new one (usecols), except for a single test to actually hit the deprecation.